cpp/keybinds/keybind_manager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#include <array>
+
+#include "keybind_manager.h"
+
+namespace openage {
+namespace keybinds {
+
+KeybindManager::KeybindManager()
+	: // TODO get this from a file instead of hardcoding it here
+	keys {{ key_t(SDLK_ESCAPE), action_t::STOP_GAME },
+		{ key_t(SDLK_F1), action_t::TOGGLE_HUD },
```

Comment:
@TheJJ did you let this one slip? 

![gnazi](https://i.imgur.com/UI0s4U3.jpg?1)